docker-compose.yml

```diff
@@ -459,6 +459,49 @@ services:
             - prosody
         networks:
             meet.jitsi:
+    # Log Analyser
+    grafana:
+        image: grafana/grafana:10.2.0
+        container_name: grafana
+        user: "0"
```

Comment:
why is this necessary?

Reply:
I removed container_name, I gave the user permission due to an error, so I used it.